File: src/transmitter.cpp
#include <ros/ros.h>
#include <std_msgs/String.h>

int main (int argc, char **argv) {
    ros::init(argc, argv, "transmitter");

    ros::NodeHandle nh;

    ros::Publisher pub = nh.advertise<std_msgs::String>("/test_message", 10);

    ros::Rate rate(3); // will ensure that while block executes at 3Hz

    while(ros::ok()) {
        std_msgs::String msg;
        msg.data = "This is test messate of the cpp publisher";
        pub.publish(msg);
        rate.sleep();
    }
}

File: scripts/transmitter.py
#!/usr/bin/env python
import rospy 
from std_msgs.msg import String

if __name__ == '__main__':
    rospy.init_node("transmitter")
    
    # message name, message type, queue
    pub = rospy.Publisher("/test_message", String, queue_size=10)

    rate = rospy.Rate(2) # Hertz

    while not rospy.is_shutdown():
        msg = String()
        msg.data = "This is test messate of the python publisher"
        pub.publish(msg)
        rate.sleep()

    rospy.loginfo("Node {} stopped".format(rospy.get_name()))